ocs_ci/utility/utils.py

```
@@ -6076,19 +6076,43 @@ def enable_huge_pages():
             node_type=constants.MASTER_MACHINE, timeout=timeout
         )
     else:
-        if num_nodes >= 6:
-            timeout = 2400
+        timeout = _mcp_timeout_for_node_count(num_nodes)
         wait_for_machineconfigpool_status(
             node_type=constants.WORKER_MACHINE, timeout=timeout
         )
 
 
+def _mcp_timeout_for_node_count(num_nodes):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_mcp_timeout_for_node_count`

Add return type annotation: `int`

(ANN202)